src/less/tooltip/base/tooltip.less

```diff
@@ -52,20 +52,22 @@ button.tooltip__close {
 
 .tooltip__pointer {
     .pointer-base();
-    .pointer-top();
```

Comment:
This is potentially going to be a breaking change to the default position of the tooltip pointer (i.e. anybody who is just using it without a modifier in skin, or without setting the `pointer` attribute in coreui)